src/compounds/legacy-product-table/src/index.tsx

```diff
@@ -557,6 +557,45 @@ const Eligibility: React.FC<EligibilityProps> = ({
   )
 }
 
+interface TelephoneInfoProps {
+  telephone?: string
```

Comment:
```suggestion
  telephone: string
```

We only render if defined below